build.js

```diff
@@ -42,11 +42,9 @@ new ClosureCompiler([
 	'--warning_level=VERBOSE',
 	'--jscomp_warning=lintChecks',
 	'--compilation_level=ADVANCED',
-	'--isolation_mode=IIFE',
-	'--js=src/trace.js',
-	'--js=src/nulldownloader.js',
-	'--js=src/nullzip.js',
-	'--js=src/nullxlsx.js',
+	'--module_resolution=NODE',
+	'--output_wrapper=(function(){%output%}).call(window);',
+	'--js=src/**.js',
 	'--js=exports/nullzip-exports.js',
 	'--js=exports/nullxlsx-exports.js',
```

Comment:
Will we still need the exports  ?

Reply:
It's for the minfied, register-to-window build